.github/workflows/cicd.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: install dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --frozen-lockfile --update-checksums
```

Comment:
why

Reply:
issue with yarn install with a new dependency with the DisTube package. 
` error https://codeload.github.com/distubejs/prism-media/tar.gz/main: Integrity checked failed for "prism-media" (none of the specified algorithms are supported)`

Getting the updated checksums resolves the integrity check failure, seemingly specifically for github package registry 
https://github.com/yarnpkg/yarn/issues/7552

Reply:
Got this same error locally after running `yarn install` and can confirm that adding that resolves it.